src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -1,20 +1,26 @@
-import PropTypes from "prop-types";
 import {observer} from "mobx-react-lite";
-import {useState} from "react";
-import ProgressDrawer from "../ProgressDrawer/ProgressDrawer";
+import React, {useState} from "react";
```

Comment:
in some files your are using { import } and some you are doing {import} does it matter or is that the linter?